draft-ietf-rats-architecture.md

```diff
@@ -1310,8 +1310,8 @@ an isolated environment might be provided by a process, a dedicated chip,
 a TEE, a virtual machine, or another secure mode of operation.
 The Attesting Environment must be protected from unauthorized modification to ensure it behaves correctly. Confidentiality protection of the Attesting Environment's signing key is vital so it cannot be misused to forge Evidence.
 
-In many cases the user or owner of a device that takes on the role of Attester must not be able to modify or extract keys from its Attesting Environments.
-For example, the owner or user of a mobile phone or FIDO authenticator might not be trusted to use the keys to report Evidence about the environment that protects the keys.
+In many cases the user or owner of a device that includes the role of Attester must not be able to modify or extract keys from the Attesting Environments.
+If access to the Attesting keys was available, then Evidence about the Attesting Environment could be forged. Some common examples include the user of a mobile phone or FIDO authenticator.
```

Comment:
```suggestion
In many cases the user or owner of a device that includes the role of Attester must not be able to modify or extract keys from the Attesting Environments to prevent creating forged Evidence.
Some common examples include the user of a mobile phone or FIDO authenticator.
```